code/datums/repositories/crew/crew.dm

```diff
@@ -78,36 +78,37 @@ var/global/datum/repository/crew/crew_repository = new()
 	var/list/tracked = list()
 	for(var/mob/living/carbon/human/H in SSmobs.mob_list)
 		if(H.wearing_rig)
-			tracked |= H.wearing_rig
+			for(var/obj/item/rig_module/healthbar/HB in H.wearing_rig.installed_modules)
```

Comment:
you can create a var on the rig to track this rather than running a loop

Reply:
Done